src/chains.jl

```diff
@@ -219,8 +219,7 @@ julia> get(chn, :param_1; flatten=true)
 (param_1 = 1,)
 ```
 """
-Base.get(c::Chains, v::Symbol; flatten = false) = get(c, [v], flatten=flatten)
```

Comment:
Moved this line down because it depends on the method below it. Makes reading easier.